deploy/vagrant/ufs/hadoop2/config_tachyon.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+namenode=$(tail -n1 /tachyon/conf/workers)
```

Comment:
sometimes I like `xxxx`, sometimes I like $(xxx).   Have formatted to `xxx` in all .sh